esbuild.js

```diff
@@ -63,13 +63,78 @@ const copyWasmFiles = {
 	},
 }
 
+const copyLocalesFiles = {
```

Comment:
I really wish this wasn't necessary - I got lost trying to figure out the best way to read in the locale files.